covalent_dispatcher/_cli/service.py

```diff
@@ -174,13 +174,9 @@ def _graceful_start(
 
     pypath = f"PYTHONPATH={UI_SRVDIR}/../tests:$PYTHONPATH" if develop else ""
     dev_mode_flag = "--develop" if develop else ""
-    no_cluster_flag = "--no-cluster"
     port = _next_available_port(port)
-    if no_cluster_flag in sys.argv:
-        launch_str = f"{pypath} python app.py {dev_mode_flag} --port {port} --no-cluster {no_cluster} >> {logfile} 2>&1"
-    else:
-        launch_str = f"{pypath} python app.py {dev_mode_flag} --port {port} >> {logfile} 2>&1"
-
+    update_config({"user_interface": {"port": port}, "dispatcher": {"port": port}})
+    launch_str = f"{pypath} python app.py {dev_mode_flag} --port {port} --no-cluster {no_cluster} >> {logfile} 2>&1"
```

Comment:
I don't think the `--no-cluster {no_cluster}` is correct

Reply:
Right! Let me try that - something about that seemed iffy